### PR TITLE
Version 1.2

### PR DESCRIPTION
**Pull request by AaronDewes**
> Link to SSH instructions in README.md

I have no clue why I wrote this commit description

### CODE REVIEW

**Comment by AaronDewes** [comment]
@Soleone This fixes the issues you pointed out in #3. I also now invited you to become a member of UmbrelInfo.

File: index.md [hunk 1]
```diff
@@ -43,6 +43,6 @@ We do not recommend running it with much money yet.
 
 ---
 
-This page is pat of [Umbrel Plus](https://UmbrelPlus.gitlab.io)
+This page is pat of [Umbrel Plus](https://UmbrelPlus.gitlab.io), a free service extending your Umbrel node.
```
Comment (louneskmt) [inline index.md:46]:
```suggestion
This page is part of [Umbrel Plus](https://UmbrelPlus.gitlab.io), a free service extending your Umbrel node.
```
Typo

As @Soleone said, you should change the name of your service.

File: troubleshooting.md [hunk 2]
```diff
@@ -123,20 +123,20 @@ sudo systemctl restart umbrel-startup
...
 ```
-~/umbrel/bin/bitcoin-cli
+docker exec bitcoin bitcoin-cli
 ```
 
 and
 
 ```
-~/umbrel/bin/lncli
+docker lncli
```
Comment (louneskmt) [inline troubleshooting.md:135]:
```suggestion
docker exec lnd lncli
```

**Review by louneskmt — CHANGES_REQUESTED** [review]

File: index.md (hunk 1 above)
Comment (Soleone) [reply index.md:46]: "Umbrel Labs" comes to mind if the goal of the service is to have beta-like features that aren't official part of Umbrel yet, but most of them will be eventually.

**Review by Soleone — APPROVED** [review]

**Comment by AaronDewes** [comment]
Replaced by #8

**Pull request closed without merging**